src/components/KpiPeriodTrend.vue

```diff
@@ -0,0 +1,212 @@
+<template>
```

Comment:
Hm, hvorfor bytte navn på denne fila? Den har fortsatt `name: 'PeriodTrendTag'`?

Hvis den må flyttes eller endre navn så er det best å gjøre det i en egen commit, for da henger git med i svingene og skjønner at det var en flytting og ikke en helt ny fil, sånn at historikken bevares.